src/database/models/SettingsModel.ts

```diff
@@ -1,15 +1,31 @@
 import {model, Schema} from 'mongoose';
 
+interface ISettings {
```

Comment:
whats the I for? what is our naming convention?

Reply:
I stands for interface